compiler/util/files.cpp

```diff
@@ -525,8 +526,23 @@ void codegen_makefile(fileinfo* mainfile, const char** tmpbinname, bool skip_com
   const char* exeExt = "";
   const char* tmpbin = "";
 
+  std::string makevar = "\0";
+
   fprintf(makefile.fptr, "CHPL_MAKE_HOME = %s\n\n", CHPL_HOME);
-  fprintf(makefile.fptr, "TMPDIRNAME = %s\n", tmpDirName);
+  fprintf(makefile.fptr, "TMPDIRNAME = %s\n\n", tmpDirName);
+
+  // Write our parsed and stored CHPL_vars as CHPL_MAKE_vars
+  for( std::map<std::string, const char*>::iterator ii=envMap.begin(); ii!=envMap.end(); ++ii)
+  {
+    makevar = (*ii).first;
```

Comment:
FYI the `->` operator is shorthand for `(*).`  Well.. that's not always true, but for what you're doing it is. [stack overflow] (http://stackoverflow.com/questions/4263796/c-pointer-dereference-operator-vs) can explain better than I can. 

i.e you can write this as `makevar = ii->first;` instead. I think there's a few places in codegen.cpp you could change this too.